content/en/blog/2023/announcing-notation-rc5.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Notation v1.0.0-RC.5 is available!
+author:  "Notation Release Team"
+date: 2023-05-18
+draft: false
+---
+
+The Notation maintainers are pleased to announce v1.0.0-RC.5 for [notation CLI](https://github.com/notaryproject/notation) and [notation-go library](https://github.com/notaryproject/notation-go). This blog walks you through the major updates in this release.
+
+## Major updates in notation and notation-go
+
+Notation is a CLI tool to sign and verify OCI artifacts. The v1.0.0-RC.5 changes include but are not limited to the following. See the [v1.0.0-RC.5 release notes](https://github.com/notaryproject/notation-go/releases/tag/v1.0.0-rc.5) for details.
+
+* Improve the output message when there is no signature associated with the image being referred
+* Add compatibility  with the [OCI distribution-spec 1.1.0-RC.2](https://github.com/opencontainers/distribution-spec/releases/tag/v1.1.0-rc2) and [OCI image-spec v1.1.0-RC.3](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0-rc.3)
+* Use SHA256 instead of insecure SHA1 for certificate thumbprint in `notation inspect`
+* Update [OCI image-spec](https://github.com/opencontainers/image-spec) from 1.1.0-RC.2 to 1.1.0-RC.3
+
+Notation-go is a collection of Golang libraries to support signing and verifying OCI artifacts. It is based on the Notary Project signature specification. Notation-go v1.0.0-RC.5 removes OCI artifact manifest type from [Sign](https://pkg.go.dev/github.com/notaryproject/notation-go@main#Sign) function.
+
+### Deprecate flag `--signature-manifest` in notation sign
+
+Since v1.0.0-RC.5, Notation no longer supports creating signatures with OCI artifact manifest type in the signing process and deprecated the flag `--signature-manifest` from `notation sign`. Please note this is a breaking change in Notation v1.0.0-RC.5. We chose to make this deprecation before the stable release due to the upstream changes in the [OCI image-spec v1.1.0-rc.3](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0-rc.3) that removed the artifact manifest.
```

Comment:
change "before the stable release" to "before v 1.0.0" release.     I say this, because "Stable" is a overloaded term, and I don't want readers to think the prior RC-s were unstable

Reply:
I changed it to v1.0.0. In fact, [Stable](https://en.wikipedia.org/wiki/Software_release_life_cycle#Stable_release) is an official term in Software release life cycle, which means the last release candidate (RC) that has passed all stages of verification and tests. The remaining bugs are considered acceptable. This release goes to [production](https://en.wikipedia.org/wiki/Deployment_environment#Production).